wiki/hgcal_ticl.md

```diff
@@ -46,13 +46,13 @@ Below is a table for some of the workflows of interest defined in CMSSW that use
 **D28** geometry. The workflows can be run with `runTheMatrix.py` along
 
 ```bash
-$ runTheMatrix.py -l 10824.5,10824.8 -j 2
+$ runTheMatrix.py -l 10824.5,10824.52 -j 2
```

Comment:
Should the workflows in these examples be something else than the pixel tracking ones? (I'd leave possible update of those to a subsequent PR though)